scripts/release/create-docs-prs.sh

```diff
@@ -0,0 +1,160 @@
+}
+trap cleanup INT TERM HUP EXIT
+
+# shellcheck source=../utils.sh
+source "$UTILS_PATH"
+
+main() {
+    # Check that version env exists
+    [ -n "$VERSION" ]
+
+    step "Update mapbox/maps-ios@publisher-staging"
+    maps_ios_upload_docs
+
+    step "Open mapbox/maps-ios@publisher-production PR"
+    maps_ios_production_docs_pr
+
+    step "Update mapbox/ios-sdk"
+    ios_sdk_update_versions
+
+    step "Open mapbox/ios-sdk PR"
+    ios_sdk_open_pr
+
+    exit 0
+}
+
+maps_ios_upload_docs() {
+    info "Checkout publisher-staging worktree"
+    local staging_docs_path="$TMP_ROOT/docs-staging"
+    git worktree add "$staging_docs_path" "publisher-staging" --quiet
+    WORKTREE_TO_REMOVE="$staging_docs_path"
+
+    git -C "$staging_docs_path" reset --hard origin/publisher-production --quiet
+
+    info "Copy docs from $DOCS_PATH"
+    cp -r "$DOCS_PATH" "$staging_docs_path/$VERSION"
+
+    pushd "$staging_docs_path"  > /dev/null || exit 1
+    git_configure_release_user
+    git config user.email "$CLA_SIGNED_GITHUB_EMAIL"
+
+    git remote set-url origin "https://x-access-token:$(mbx-ci github writer public token)@github.com/mapbox/mapbox-maps-ios.git"
+
+    info "Commit"
+    git add .
+    git commit -m "Add documentation for v$VERSION" --quiet
+
+    info "Push"
+    git push --force --quiet
+
+    popd > /dev/null
+}
+
+maps_ios_production_docs_pr() {
+    local body="Update docs for v$VERSION
+[Staging version](https://docs.tilestream.net/ios/maps/api/$VERSION/)"
+
+    PRODUCTION_DOCS_PR_URL=$(GITHUB_TOKEN=$(mbx-ci github writer public token) \
+        gh pr create --repo mapbox/mapbox-maps-ios \
+            --head "publisher-staging" --base "publisher-production" \
+            --draft \
+            --title "Production docs for \`v$VERSION\`" \
+            --body "$body" \
+            --label "docs :scroll:")
+
+    info "New PR: $PRODUCTION_DOCS_PR_URL"
+}
+
+ios_sdk_update_versions() {
+    IOS_SDK_REPO_PATH="$TMP_ROOT/ios-sdk"
+
+    info "Clone repository"
+    [[ -d $IOS_SDK_REPO_PATH ]] && rm -rf "$IOS_SDK_REPO_PATH"
+    git clone "https://x-access-token:$(mbx-ci github writer private token)@github.com/mapbox/ios-sdk.git" "$IOS_SDK_REPO_PATH" --branch publisher-production --depth=1 --quiet
+
+    cd "$IOS_SDK_REPO_PATH" || exit 1
+
+    IOS_SDK_BRANCH_NAME="maps-sdk/ios/${VERSION}"
+
+    git_configure_release_user
+
+    git checkout -b "${IOS_SDK_BRANCH_NAME}" --quiet
+
+    # Apply only for release versions
+    if [[ "$VERSION" =~ ^v?[0-9]*\.[0-9]*\.[0-9]*$ ]]; then
+        info "Update release version"
+        # shellcheck disable=SC2005 # This solution is intentional to modify json in one line with jq
+        echo "$(jq ".VERSION_IOS_MAPS_SDK = \"$VERSION\"" src/constants.json)" > src/constants.json
+    fi
+
+    info "Add version to the list"
+    # shellcheck disable=SC2005
+    echo "$(jq "[\"${VERSION}\"] + ." src/data/ios-maps-sdk-versions.json)" > src/data/ios-maps-sdk-versions.json
```

Comment:
Does it matter if we keep the array sorted?

Reply:
But we do not enforce sorted array here, right?

Reply:
The array in that file is sorted at the moment. I believe it's the source for the list at https://docs.mapbox.com/ios/maps/api-reference/. If we scramble it, would the content on that page be scrambled too, or is there some sorting that happens when that array is consumed? @colleenmcginnis could you advise?

Reply:
I've added sorting logic (implemented by `sort -rV`) but I'm not sure is it okay.
Here is the changeset sample:
https://github.com/mapbox/ios-sdk/commit/f5f0f1609445000660e64ba1e7111e88419088fa?diff=split

Reply:
We have logic in the component that renders the versions on the page that will sort versions automatically so it doesn't matter what order versions are in in the array.

Reply:
Thanks @colleenmcginnis. @OdNairy sounds like we don't need to worry about it!